src/main/java/com/smalaca/messagesender/controller/UserController.java

```diff
@@ -11,25 +11,24 @@
 @RequestMapping("/user")
 @RestController
 public class UserController {
-    public static final String FORBIDDEN_LOGIN = "";
 
     @Autowired
     private UserCrud userCrud;
 
     @RequestMapping("/create")
-    public ResponseEntity<String> createUser(@ModelAttribute UserDto userDto) {
+    public String createUser(@ModelAttribute UserDto userDto) {
         if (userCrud.createUser(userDto))
-            return new ResponseEntity<>("User Created", HttpStatus.OK);
+            return "User Created";
         else
-            return new ResponseEntity<>("User Creation Failed!", HttpStatus.OK);
+            return "User Creation Failed!";
     }
 
     @RequestMapping("/block")
-    public ResponseEntity<String> blockUser(@RequestParam(value = "login", defaultValue = FORBIDDEN_LOGIN) String login) {
-        if (login.equals(FORBIDDEN_LOGIN)) {
+    public ResponseEntity<String> blockUser(@ModelAttribute UserDto userDto) {
+        if (userDto.getLogin().equals("")) {
```

Comment:
Polecam poczytać o [Tell, don't ask](https://martinfowler.com/bliki/TellDontAsk.html) oraz [enkapsulacji](https://pl.wikipedia.org/wiki/Hermetyzacja_(informatyka)) 😄 

Byłoby lepiej:
```if (userDto.hasLogin()) {```
Wtedy wiesz o co pytasz. To w jaki sposób odpowiadasz na pytanie jest mniej istotne w tym momencie.

Reply:
A czy w ten sposób nie dodajemy dodatkowych i nadmiarowych/zbędnych odpowiedzialności do UserDto? Miał mieć tylko pola, gettery i settery.